pkgs/data/fonts/u001/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation rec {
```

Comment:
Not sure about this, but wouldn't it be better to use `stdenvNoCC`?
```suggestion
stdenvNoCC.mkDerivation rec {
```

Reply:
I did not know about this. Updated.